src/selectedtests/git_helper.py

```diff
@@ -0,0 +1,79 @@
+"""Git helper for mappings commands."""
+import os.path
+import structlog
+
+from typing import Any, Set
+from git import Commit, Diff, DiffIndex, Repo
+from structlog.stdlib import LoggerFactory
+
+
+structlog.configure(logger_factory=LoggerFactory())
+GITHUB_BASE_URL = "https://github.com"
+
+
+def init_repo(temp_dir, repo_name: str, branch: str, org_name: str = "mongodb") -> Repo:
+    """
+    Create the given repo in the given directory and checkout the given branch.
+
+    :param temp_dir: The place where to clone the repo to.
+    :param repo_name: The name of the repo to clone.
+    :param branch: The branch to checkout in the repo.
+    :param org_name: The org name in github that owns the repo.
+    :return: An Repo instance that further git operations can be done on.
+    """
+    repo_path = os.path.join(temp_dir, repo_name)
+    url = f"{GITHUB_BASE_URL}/{org_name}/{repo_name}.git"
+    repo = Repo.clone_from(url, repo_path, branch=branch)
+    return repo
+
+
+def _paths_for_iter(diff: Diff, iter_type: str):
```

Comment:
Can you add some tests for this?

Reply:
Added some tests for the public `modified_files_for_commit` method which calls this